docs/UserGuide.md

```diff
@@ -90,90 +91,147 @@ interactions recorded faster and simpler than traditional GUI applications.
 
 ### Adding a client: `add`
 
-Adds an empty client to the list without any Remark and transactions.
+Adds a client to the list.
 
-Format: `add n/NAME a/ADDRESS p/PHONE e/EMAIL [t/TAG]`
+Format: `add n/NAME a/ADDRESS p/PHONE e/EMAIL [t/TAG]...`
 
 * Tags are optional.
 * Multiple tags can be tagged to the client.
-* Use `create` command to add Remark to the client.
 
 Examples:
-* `add n/MacDonalds a/West Coast Park p/9876542 e/mcd@mcd.com`
-* `add n/KFC a/Yishun Street 81 p/9876543 e/kfc@kfc.com t/FriedChicken`
+* `add n/Alice a/West Coast Park p/9876542 e/alice@gmail.com`
+* `add n/John a/Yishun Street 81 p/9876543 e/john@yahoo.com t/friends t/supplier`
+
+### Creating a Transaction: `buy` or `sell`
+
+Creates a  `buy` or `sell` transaction linked to a client.
+
+Formats:
+
+`buy INDEX q/QUANTITY g/GOODS price/PRICE [d/DATE]`
+
+`sell INDEX q/QUANTITY g/GOODS price/PRICE [d/DATE]`
+
+* The `INDEX` refers to the index number shown in the displayed client list.
+* The `INDEX` **must be a positive integer** 1, 2, 3, …​
+* The `QUANTITY` refers to the amount of the goods transacted.
+* The `QUANTITY` should only contain numbers and be at least 1 digit long.
```

Comment:
Great implementation! Maybe can state that Quantity must be Non-Zero.

Reply:
sure will add this in